js/babel-plugin-sprockets-commoner-internal/index.js

```diff
@@ -25,7 +25,9 @@ if (typeof Object.assign != 'function') {
 
 var fs = require('fs');
 var dirname = require('path').dirname;
+var join = require('path').join;
 var resolve = require('browser-resolve').sync;
+var emptyModule = join(__dirname, 'node_modules', 'browser-resolve', 'empty.js');
```

Comment:
Maybe we can just check if the resolved path is inside the `browser-resolve` package? Makes me a bit worried to point at a particular file.